modules/python3/bindings/src/pydatareaders.cpp

```diff
@@ -117,26 +98,18 @@ void exposeFactoryReaderType(pybind11::class_<DataReaderFactory>& r, std::string
 void exposeDataReaders(pybind11::module& m) {
     namespace py = pybind11;
 
-    py::class_<DataReader, DataReaderTrampoline>(m, "DataReader")
-        .def("clone", &DataReader::clone)
-        .def_property_readonly("extensions", &DataReader::getExtensions,
-                               py::return_value_policy::reference_internal)
-        .def("addExtension", &DataReader::addExtension)
-        .def("setOption", &DataReader::setOption)
-        .def("getOption", &DataReader::getOption);
-
```

Comment:
Why was the base class removed?

Reply:
I was assuming that it will not be used as the base class functionality are exposed for DataReaderType anyway. Perhaps there is such as use case though?

Reply:
How is the base class exposed in the DataReaderType? You just removed exactly that?

Reply:
Otherwise those functions will not be exposed in the typed readers

Reply:
This part should still be in there? if not pls explain, it is only the DataReaderTrampoline that should be removed.